.travis.yml

```diff
@@ -62,6 +62,19 @@ script:
   - make -j4 ; sudo make install
   - make check
 
+jobs:
+  include:
+    - os: linux
+    - os: freebsd
+      compiler: clang
+      before_install:
+        - sudo pkg install -y meson pkgconf libdrm libXext libXfixes wayland
+        - sudo pkg install -y -x '^mesa($|-libs)'
+      script:
+        - meson build
+        - ninja -C build
+      after_success: true
```

Comment:
`true` here because empty values produces errors:
```
/home/travis/build.sh: line 1436: travis_run_after_success: command not found
/home/travis/build.sh: line 1437: travis_run_after_failure: command not found
/home/travis/build.sh: line 1438: travis_run_after_script: command not found
/home/travis/build.sh: line 1439: travis_run_finish: command not found
```